web/handler/pizzas.ml

```diff
@@ -1,8 +1,49 @@
 let index req =
   let open Lwt.Syntax in
+  let csrf = Sihl.Web.Csrf.find req |> Option.get in
+  let alert = Sihl.Web.Flash.find_alert req in
+  let notice = Sihl.Web.Flash.find_notice req in
   let* user = Service.User.Web.user_from_session req |> Lwt.map Option.get in
-  Lwt.return @@ Sihl.Web.Response.of_html (View.Pizzas.index user)
+  let* pizzas = Pizza.find_pizzas () in
+  Lwt.return
+  @@ Sihl.Web.Response.of_html
+       (View.Pizzas.index user ~alert ~notice csrf pizzas)
+;;
+
+let show req =
+  let open Lwt.Syntax in
+  let csrf = Sihl.Web.Csrf.find req |> Option.get in
+  let alert = Sihl.Web.Flash.find_alert req in
+  let notice = Sihl.Web.Flash.find_notice req in
+  let* user = Service.User.Web.user_from_session req |> Lwt.map Option.get in
+  let name = Sihl.Web.Router.param req "name" in
+  let* pizza = Pizza.find_pizza name in
+  match pizza with
+  | None ->
+    Sihl.Web.Response.redirect_to "/pizzas"
+    |> Sihl.Web.Flash.set_notice (Format.sprintf "Pizza '%s' not found" name)
+    |> Lwt.return
+  | Some pizza ->
+    Lwt.return
+    @@ Sihl.Web.Response.of_html
+         (View.Pizzas.show user ~alert ~notice csrf pizza)
 ;;
 
 let create _ = failwith "todo pizza create"
```

Comment:
@timohuber Since you added a `create_form` in the view, could you also wire this up so we could create pizzas? You will have to extend the `create_form` so it also has an input field for ingredients. You could make it so the input takes all ingredients comma separated (with a hint for that below the input field) and then you parse it in the backend and turn it into a list. You can then use https://github.com/oxidizing/sihl-demo/blob/main/app/context/pizza/pizza.ml#L44 to create the pizza.